### PR TITLE
Qstr qml v2

### PR DESCRIPTION
**Pull request by tcanabrava**
this version of the qt-translation system for the qml files fixes a few merge conflicts.

### CODE REVIEW

File: src/FlightDisplay/FlightDisplayViewWidgets.qml [hunk 1]
```diff
@@ -391,21 +391,21 @@ Item {
...
                 altitudeSlider.setInitialValueMeters(10)
-                guidedModeConfirm.confirmText = "takeoff"
+                guidedModeConfirm.confirmText = qsTr("takeoff")
                 break;
             case confirmLand:
-                guidedModeConfirm.confirmText = "land"
+                guidedModeConfirm.confirmText = qsTr("land")
                 break;
             case confirmHome:
                 guidedModeConfirm.confirmText = "return to launch"
```
Comment (DonLakeFlyer) [inline src/FlightDisplay/FlightDisplayViewWidgets.qml]: Missed one?

File: src/FlightMap/Widgets/VibrationWidget.qml [hunk 2]
```diff
@@ -84,7 +84,7 @@ QGCFlickable {
                 QGCLabel {
                     id:     xBarLabel
                     color:  textColor
-                    text:   "X"
+                    text:   qsTr("X")
```
Comment (DonLakeFlyer) [inline src/FlightMap/Widgets/VibrationWidget.qml]: This is X/Y/Z axis. Seems like this shouldn't be translated? Not sure.

**Comment by DonLakeFlyer** [comment]
Could you take a quick run through the setup pages in both PX4 and APM mocklink connections just to make sure nothing strange happens?

**Comment by tcanabrava** [comment]
Sure. will run the setup and brb.
about the 'x' and 'y' axys, I don't know if in chinese or japanese they
have a symbol for that, that's why I marked it for translation.
and yes, I missed one, will update in a moment.

**Comment by DonLakeFlyer** [comment]
I’d leave off X/Y/Z axis names from translation

**Comment by DonLakeFlyer** [comment]
@tcanabrava Can you finish the tweaks on this one so we can merge?

**Comment by tcanabrava** [comment]
Yes, sorry. Focused on other things this week
Em 2 de abr de 2016 14:44, "Don Gagne" <notifications@github.com> escreveu:

**Pull request closed without merging**